### DISCUSSION TITLE
Web Vault and DUO - expired authorization

### DISCUSSION THREAD

**Discussion by natrlhy** [dani-garcia/vaultwarden#1757] [Q&A] [answered]
Hello!

I was trying to login to my web vault and I am getting my DUO push authorization and it's accepted, however right when I approve it, Bitwarden (vaultwarden) gives me an error:

An error has occurred: Expired authorization

![image](https://user-images.githubusercontent.com/19696424/121093940-a6ae4400-c7a2-11eb-944c-12efd269a576.png)

I just upgraded to the latest version and still see this issue.
I was hoping that someone could help me get past this as I'm unable to login to my vault since I have DUO 2FA setup on this vault.

Thanks!

UPDATE:

I just created another account, enabled DUO 2FA on it as well and get the same error. Not seeing an issue with DUO during this process...

**Comment by BlackDex** [comment, marked as answer]
You probably have an ad blocker installed as extension. Or very high privacy settings which block/remove the local storage of session info. Or the time and date is wrong on the server or client.

**Reply by natrlhy** [reply]
Sweet! I checked the vaultwarden time on my synology server and saw it was about 1 minute ahead (it's configured to sync with ntp time to time.google.com)

Also the admin page saw this:

```
Date & Time (Local)
    Server: 2021-06-08 06:32:29 +00:00 
Date & Time (UTC) Ok
    Server: 2021-06-08 06:32:29 UTC
    Browser: 2021-06-08 06:31:10 UTC
```
I was seeing the issue on all browsers (with and without extensions enabled).

Doing an ntp sync fixed the issue!

Thank you! 

How does one set the timezone on the container?
